hcatalog/core/src/main/java/org/apache/hive/hcatalog/mapreduce/InternalUtil.java

```diff
@@ -143,18 +142,16 @@ private static ObjectInspector getObjectInspector(TypeInfo type) throws IOExcept
   //TODO this has to find a better home, it's also hardcoded as default in hive would be nice
   // if the default was decided by the serde
   static void initializeOutputSerDe(AbstractSerDe serDe, Configuration conf, OutputJobInfo jobInfo)
-    throws SerDeException {
-    SerDeUtils.initializeSerDe(serDe, conf,
-                               getSerdeProperties(jobInfo.getTableInfo(),
-                                                  jobInfo.getOutputSchema()),
-                               null);
+      throws SerDeException {
+    serDe.initialize(conf, getSerdeProperties(jobInfo.getTableInfo(), jobInfo.getOutputSchema()), null);
   }
 
   static void initializeDeserializer(Deserializer deserializer, Configuration conf,
                      HCatTableInfo info, HCatSchema schema) throws SerDeException {
     Properties props = getSerdeProperties(info, schema);
     LOG.info("Initializing " + deserializer.getClass().getName() + " with properties " + props);
-    SerDeUtils.initializeSerDe(deserializer, conf, props, null);
+    AbstractSerDe serde = (AbstractSerDe)deserializer;
```

Comment:
Nit: please call this serDe for consistency.

Reply:
Done.